shell/assets/translations/en-us.yaml

```diff
@@ -4613,6 +4613,7 @@ rbac:
       restricted-admin:
         label: Restricted Administrator
         description: Restricted Admins have full control over all resources in all downstream clusters but no access to the local cluster.
+        deprecation: 'Warning: The Restricted Administrator role has been deprecated as of Rancher 2.8.0 and will be removed in a future release - Check out the <a href="{releaseNotesUrl}" target="_blank" rel="noopener noreferrer nofollow">Release Notes</a>'
```

Comment:
The target blank doesn't seem to work - it opens in the existing tab - not sure what is wrong.

Reply:
It was the `v-clean-html` directive that was sanitizing it. Check if you agree with my change on the PR (added a hook to put the `target` back on links if `rel` property is well populated.